pkg/test/framework/settings/flags.go

```diff
@@ -18,62 +18,40 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"strings"
-
-	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
 
 	"istio.io/istio/pkg/log"
 )
 
-var settings = defaultSettings()
-var showHelp = false
-var logOptions = log.DefaultOptions()
-
-// The command we use to parse flags.
-var cmdForLog = &cobra.Command{}
+var (
+	settings   = defaultSettings()
+	logOptions = log.DefaultOptions()
```

Comment:
Can you add a TODO here to add regular flags for the sake of log options?

This has been quite useful for debugging code that runs local/in-memory.

Reply:
done.